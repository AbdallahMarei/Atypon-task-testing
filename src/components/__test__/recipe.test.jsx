import { shallow,mount } from 'enzyme';
import "../../setupTest";
import Recipe from '../recipe/recipe'


const loading = false;

describe('Recipe', () => {
    it('renders correctly', () => {
        const wrapper = shallow(<Recipe />);
        expect(wrapper).toMatchSnapshot();
    });
    it('accepts loading props', ()=>{
        const wrapper = mount(<Recipe loading={loading} />);
        expect(wrapper.props().loading).toEqual(loading)
    })
});