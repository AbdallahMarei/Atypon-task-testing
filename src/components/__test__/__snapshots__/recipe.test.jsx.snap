// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Recipe renders correctly 1`] = `ShallowWrapper {}`;
